src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -166,6 +249,8 @@ export const AppLevelExternalLinks = ({
     }, [externalLinks, monitoringTools])
 
     const getExternalLinkChip = (linkOption: OptionTypeWithIcon, idx: number) => {
+        const externalLinkURL = getParsedURL(true, linkOption.value.toString(), details)
+        const handleTextClick = () => handleOpenModal(linkOption, externalLinkURL)
```

Comment:
Ideally we should not define method inside method, please create a component if possible